modules/test/playwright/tests/object-web/main/objectEntries.spec.ts

```diff
@@ -2599,6 +2599,72 @@ test.describe('Manage object entries through View Object Entries', () => {
 		}
 	);
 
+	test('verify that API is invoked only once', async ({
+		apiHelpers,
+		page,
+		viewObjectEntriesPage,
+	}) => {
+		const objectFields = generateObjectFields({
+			objectFieldBusinessTypes: ['Text'],
+		});
+
+		const objectDefinition =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition({
+				objectFields,
+				status: {code: 0},
+			});
+
+		apiHelpers.data.push({
+			id: objectDefinition.id,
+			type: 'objectDefinition',
+		});
+
+		const objectRelationshipAPIClient = await apiHelpers.buildRestClient(
+			ObjectRelationshipAPI
+		);
+
+		await objectRelationshipAPIClient.postObjectDefinitionByExternalReferenceCodeObjectRelationship(
+			'L_ACCOUNT',
+			{
+				label: {
+					en_US: 'objectRelationshipLabel' + getRandomInt(),
+				},
+				name: 'objectRelationshipName' + getRandomInt(),
+				objectDefinitionExternalReferenceCode1: 'L_ACCOUNT',
+				objectDefinitionExternalReferenceCode2:
+					objectDefinition.externalReferenceCode,
+				type: 'oneToMany',
+			}
+		);
+
+		let apiInvoked = 0;
+
+		page.on('request', (request) => {
+			if (
+				request
+					.url()
+					.includes('/o/headless-admin-user/v1.0/accounts') &&
+				request.method() === 'GET'
+			) {
+				apiInvoked++;
+			}
+		});
+
+		await viewObjectEntriesPage.goto(objectDefinition.className);
+
+		await viewObjectEntriesPage.addObjectEntryButton.click();
+
+		await page.waitForResponse(
+			(request) =>
+				request
+					.url()
+					.includes('/o/headless-admin-user/v1.0/accounts') &&
+				request.request().method() === 'GET'
+		);
```

Comment:
Since we're already tracking the page requests conditionally, I believe this is not needed! can you check? thanks!